### PR TITLE
Parsecontext rpt

### PR DESCRIPTION
**Pull request by joakim-hove**
We now handle:

```
RPTRST
   BASIC = 2 /
```
i.e. unquoted spaces around the "=". Regulated by the error mode `RPT_MIXED_STYLE` which defaults to `WARN`.

@alfbr: Please give this a spin and merge if appropriate.

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

File: opm/parser/eclipse/Parser/ParseContext.hpp [hunk 1]
```diff
@@ -264,14 +264,46 @@ namespace Opm {
...
+        const static std::string RPT_UNKNOWN_MNEMONIC;
+
+
+    private: void initDefault(); void initEnv(); void envUpdate( const
+        std::string& envVariable , InputError::Action action ); void
+        patternUpdate( const std::string& pattern , InputError::Action action);
         std::map<std::string , InputError::Action> m_errorContexts;
-        std::set<std::string> ignore_keywords;
-}; }
+        std::set<std::string> ignore_keywords; }; }
```
Comment (bska) [inline opm/parser/eclipse/Parser/ParseContext.hpp]: That's an&mdash;ehm&mdash;avant-garde stylistic choice.

Comment (joakim-hove) [reply opm/parser/eclipse/Parser/ParseContext.hpp]: That was an editor xxx

Comment (bska) [reply opm/parser/eclipse/Parser/ParseContext.hpp]: Did you actually **merge** that abomination?  Do you think
```C++
    private: void initDefault(); void initEnv(); void envUpdate( const
        std::string& envVariable , InputError::Action action ); void
        patternUpdate( const std::string& pattern , InputError::Action action)
        std::map<std::string , InputError::Action> m_errorContexts;
        std::set<std::string> ignore_keywords; }; }
```
is reasonable formatting?  If so, I'd very much like to hear the reasoning behind it.

Comment (joakim-hove) [reply opm/parser/eclipse/Parser/ParseContext.hpp]: Yes I did - it was a mistake; I will fix it now.

Comment (joakim-hove) [reply opm/parser/eclipse/Parser/ParseContext.hpp]: #599

**Pull request merged**